tests/datamodules/test_urban3d.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I'm trying to avoid adding new tests in `tests/datamodules` and instead adding configuration files and a single line to `tests/trainers` to test the datamodule with a real trainer. This tends to be a much better test of the datamodule.